engines/macventure/gui.cpp

```diff
@@ -844,7 +895,7 @@ void Gui::updateExit(ObjID obj) {
 	if (!_engine->isObjExit(obj)) {
 		return;
 	}
-	BorderBounds border = borderBounds(getWindowData(kExitsWindow).type);
+	//BorderBounds border = borderBounds(getWindowData(kExitsWindow).type);
```

Comment:
same here and below - remove the commented code - that's basically what the version control system is giving us already.